credentials/IvanIvanov.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
is it part of fix - looks no.
 Next time, let's separate 'other commits' to their own pull-requests